src/recipes/editor-plugin/inspector-plugins.md

```diff
@@ -0,0 +1,343 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Inspector plugins
+
+The inspector dock allows you to create custom widgets to edit properties through plugins.
+This can be beneficial when working with custom datatypes and resources, although you can
+use the feature to change the inspector widgets for built-in types. You can design custom
+controls for specific properties, entire objects, and even separate controls associated
+with particular datatypes.
+
+This is implementation of
+[Godot docs](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/inspector_plugins.html) on Rust.
+
+Before:
+
+![Before](./images/before.png)
+
+After:
+
+![After](./images/after.png)
+
+Add requires to Rust (in Rust folder, where is Cargo.toml):
+
+```bash
+cargo add rand
+```
+
+Add file `addon.rs` and import it in `lib.rs`:
+
+```rust
+// file: lib.rs
+mod addon;
+```
+
+Add the following imports at the beginning of the file
+
+```rust
+use godot::{
+    classes,
+    engine::{
+        Button, EditorInspectorPlugin, EditorPlugin, EditorProperty, IEditorInspectorPlugin,
+        IEditorPlugin, IEditorProperty,
+    },
+    global,
+    prelude::*,
+};
+use rand::Rng;
+```
+
+Since Rust is a statically typed language, we will proceed in reverse order unlike in Godot documentation, to avoid encountering errors unnecessarily.
```

Comment:
Not sure this line is necessary since people going through this tutorial probably won't be going through the Godot one at the same time.  But you can leave it if you prefer.

Reply:
Just for show why it has upside down flow)